java/client/src/main/java/glide/api/commands/GenericBaseCommands.java

```diff
@@ -542,4 +547,46 @@ CompletableFuture<Boolean> pexpireAt(
      * }</pre>
      */
     CompletableFuture<Long> touch(String[] keys);
+
+    /**
+     * Copies the value stored at the <code>source</code> to the <code>destination</code> key.
+     *
+     * @since Redis 6.2.0 and above.
+     * @apiNote When in cluster mode, both <code>source</code> and <code>destination</code> must map
+     *     to the same hash slot.
+     * @see <a href="https://redis.io/commands/copy/">redis.io</a> for details.
+     * @param source The key to the source value.
+     * @param destination The key where the value should be copied to.
+     * @param replace If the destination key should be removed before copying the value to it.
+     * @return <code>1L</code> if <code>source</code> was copied, <code>0L</code> if <code>source
+     *     </code> was not copied.
+     * @example
+     *     <pre>{@code
+     * client.set("test1", "one").get();
+     * client.set("test2", "two").get();
+     * assertEquals(0L, client.copy("test1", "test2", false).get());
+     * assertEquals(1L, client.copy("test1", "test2", true).get());
+     * }</pre>
+     */
+    CompletableFuture<Long> copy(String source, String destination, boolean replace);
```

Comment:
copy should return a boolean
value_conversion.rs has a convert to boolean return value for commands that return 0/1